Frontend_Web_Development_React_Angular_Vue/React/Week 2/Components.md

```diff
@@ -95,7 +95,7 @@ The output will be:
 Hello World!
 ```
 
-2. `class` components - class components are written in ES6 classes. Every class component extends `React.Component`. Every class component must have a render method that returns some sort of JSX / markup / call to `React.createElement`.
+2. `class` components - class components are written in ES6 classes. We can make our own class and inherit from that class, to be able to use the methods defined in that parent class. Every class component must have a render method that returns some sort of JSX / markup / call to `React.createElement`.
```

Comment:
In general applications, we inherit from React.Component. But, we can also make our own class and inherit from that class, to be able to use the methods defined in that parent class.